htdocs/install/mysql/tables/llx_c_field_list.sql

```diff
@@ -33,6 +33,7 @@ create table llx_c_field_list
   align			varchar(6)		DEFAULT 'left',				-- align (left,center,right)
   sort			tinyint 		DEFAULT 1  	NOT NULL,		-- add sort field
   search		tinyint 		DEFAULT 0  	NOT NULL,		-- add search field
+  checked		tinyint 		DEFAULT 0  	NOT NULL,		-- show or hide field by default
```

Comment:
It would be great to use same name and values that the one use for php array declaration of fields with the new module builder template (it is also used by extrafields) so a field liek this : 

visible"    tinyint  DEFAULT 1  NOT NULL,  -- visibility of field. 0=Never visible, 1=Visible on list and forms, 2=Visible on list only. Using a negative value means field is not shown by default on list but can be selected for viewing

Reply:
@eldy 
ok it's fixed